virttest/ceph.py

```diff
@@ -167,7 +166,7 @@ def rbd_image_map(ceph_monitor, rbd_pool_name, rbd_image_name):
     """
     cmd = "rbd map %s --pool %s -m %s" % (rbd_image_name, rbd_pool_name,
                                           ceph_monitor)
-    output = decode_to_text(process.system_output(cmd, verbose=True))
+    output = process.system_output(cmd, verbose=True).stdout_text
```

Comment:
I notice that previous process.system_output was replaced by process.run(), so  maybe we can keep it persistent